v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/BigtableChangeStreamsToGcsOptions.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.options;
+
+import com.google.cloud.teleport.metadata.TemplateParameter;
+import com.google.cloud.teleport.v2.transforms.WriteChangeStreamMutationToGcsAvro;
+import com.google.cloud.teleport.v2.transforms.WriteChangeStreamMutationsToGcsText;
+import com.google.cloud.teleport.v2.utils.WriteToGCSUtility.BigtableSchemaFormat;
+import com.google.cloud.teleport.v2.utils.WriteToGCSUtility.FileFormat;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Validation;
+
+/**
+ * The {@link BigtableChangeStreamsToGcsOptions} interface provides the custom execution options
+ * passed by the executor at the command-line.
+ */
+public interface BigtableChangeStreamsToGcsOptions extends BigtableChangeStreamsToGcsFilterOptions {
+    @TemplateParameter.ProjectId(
+        order = 1,
+        optional = true,
+        description = "Bigtable Project ID",
+        helpText =
+            "Project to read change streams from. The default for this parameter is the project "
+                + "where the Dataflow pipeline is running.")
+    @Default.String("")
+    String getBigtableProjectId();
+
+    void setBigtableProjectId(String projectId);
+
+    @TemplateParameter.Text(
+            order = 2,
+            description = "Bigtable Instance ID",
+            helpText = "The Bigtable Instance to read change streams from.")
+    @Validation.Required
+    String getBigtableInstanceId();
+
+    void setBigtableInstanceId(String bigtableInstanceId);
+
+    @TemplateParameter.Text(
+            order = 3,
+            description = "Bigtable Table ID",
+            helpText = "The Bigtable table ID to read change streams from.")
+    @Validation.Required
+    String getBigtableTableId();
+
+    void setBigtableTableId(String bigtableTableId);
+
+    @TemplateParameter.Text(
+        order = 4,
+        description = "Bigtable App Profile ID",
+        helpText = "The Bigtable App Profile ID to read change streams from.")
+    @Validation.Required
+    String getBigtableAppProfileId();
```

Comment:
Should we call is getBigtableAppProfile or getBigtableAppProfileId ?

Reply:
I chose (getBigtable)appProfileId following what was done in the beam connector: 
https://team.git.corp.google.com/cloud-bigtable-team/cdc-beam/+/refs/heads/staging/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java#1583